conf/base.py

```diff
@@ -152,6 +152,14 @@
     'TEST_REQUEST_DEFAULT_FORMAT': 'vnd.api+json',
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.URLPathVersioning',
     'COERCE_DECIMAL_TO_STRING': False,
+    'DEFAULT_THROTTLE_CLASSES': (
```

Comment:
Throttle classes should only be enabled if settings.PROFILES_ENABLED is true

Reply:
Done